lib/fcdpp_control_impl.cc

```diff
@@ -97,18 +97,18 @@ void fcdpp_control_impl::set_freq(float freq)
         nfreq += (unsigned int)(aucBuf[3] << 8);
         nfreq += (unsigned int)(aucBuf[4] << 16);
         nfreq += (unsigned int)(aucBuf[5] << 24);
-        GR_LOG_INFO(d_logger, boost::format("Set Frequency to: %1% Hz") % nfreq);
+        d_logger->info("Set Frequency to: {:d} Hz", nfreq);
 
     } else {
-        GR_LOG_INFO(d_logger, boost::format("Set Frequency to %1% Hz failed") % nfreq);
+        d_logger->error("Set Frequency to {:d} Hz failed", nfreq);
```

Comment:
I assumed that this message was meant to be logged at the `error` level, so I changed that here.